WIT.md

```diff
@@ -44,16 +44,41 @@ whitespace ::= ' ' | '\n' | '\r' | '\t'
 
 ### Comments
 
-A `comment` token in `*.wit` is either a line comment preceded with `//` which
-ends at the next newline (`\n`) character or it's a block comment which starts
+Commenting your code might have two purposes: implementation details and documentation. 
+Therefore, there are two types of comments for each purpose. 
+
+#### Implementation comments
+
+An `implementation comment` token in `*.wit` is either a line comment preceded with `//` 
+which ends at the next newline (`\n`) character or it's a block comment which starts
 with `/*` and ends with `*/`. Note that block comments are allowed to be nested
 and their delimiters must be balanced
 
 ```wit
-comment ::= '//' character-that-isnt-a-newline*
-          | '/*' any-unicode-character* '*/'
+implementation-comment ::= '//' character-that-isnt-a-newline*
+                         | '/*' any-unicode-character* '*/'
 ```
 
+#### Documentation comment
+
+A `documentation comment` token in `*.wit` is a documentation line preceded with `///` 
+which ends at the next newline (`\n`) character or it's a block doc comment which starts
+with `/**` and ends with `**/`. Note that block doc comments are allowed to be nested
```

Comment:
To match the Rust style I think the ending here will be `*/` because I believe the intention is that doc comments are subsets of normal comments.

Reply:
True! My bad, I usually do not use block comments. I will change depending on the rest of the review.